vit-servicing-station-lib/src/v0/endpoints/proposals/requests.rs

```diff
@@ -0,0 +1,9 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct ProposalVoteplanIdAndIndex {
```

Comment:
I think we usually write `VotePlanId` in chain-libs, so I find the capitalization here a bit odd (for the `p`)

Reply:
Actually I need to check how is this returned in the API. Good point.